### DISCUSSION TITLE
ui   matrix loaders

### DISCUSSION THREAD

**Discussion by gitjxm** [apache/dolphinscheduler#15265] [Q&A] [answered]
Failed to compile.

./node_modules/ml-matrix/src/symmetricMatrix.js 8:2
Module parse failed: Unexpected character '#' (8:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export class SymmetricMatrix extends AbstractMatrix {
|   /** @type {Matrix} */
>   #matrix;
| 
|   get size() {

**Comment by JLHwung** [comment]
See https://github.com/babel/babel-loader/issues/1012#issuecomment-1839799674

**Comment by Wizdd** [comment]
Same problem. Is it fixed now?

**Comment by SbloodyS** [comment, marked as answer]
Try upgrade these packages in package.json.
```
"@antv/layout": "^0.3.25",
"echarts": "5.5.0"
```
